exasol_integration_test_docker_environment/lib/test_environment/create_certificates/analyze_certificate_container.py

```diff
@@ -82,7 +87,7 @@ def get_goals(self):
 
 class DockerCertificateContainerBuild(DockerCertificateBuildBase):
 
-    def run_task(self):
+    def run_task(self) -> Iterator[DockerCreateImageTask]:
```

Comment:
`DockerCreateImageTask` is not 100% correct, as it can be different classes. Maybe in another PR...